README.rst

```diff
@@ -132,10 +132,10 @@ data (see `package_show_version`_)
 **Example**::
 
   $ curl -H "Authorization: $API_KEY" \
-    https://ckan.example.com/api/3/action/dataset_version_show?id=5942ab7a-67cb-426c-ad99-dd4519530bc7
+    https://ckan.example.com/api/3/action/dataset_tag_show?id=5942ab7a-67cb-426c-ad99-dd4519530bc7
```

Comment:
Is still correct? The parameter for showing a tag is `id` and the expected value is a UUID?

Reply:
The docs are a little bit outdated and will require some love after all the refactor is finished :)

Reply:
Also `dataset_tag_show` is a very confusing API name. It should be something like `dataset_revision_tag`